README.md

```diff
@@ -105,9 +111,13 @@ you should look into also using [clue/reactphp-block](https://github.com/clue/re
 The resulting blocking code could look something like this:
 
 ```php
+<?php
+
+require __DIR__ . '/vendor/autoload.php';
+
 use Clue\React\Block;
 
-$factory = new Factory();
+$factory = new Clue\React\Mdns\Factory();
 $resolver = $factory->createResolver();
 
 $promise = $resolver->lookup('me.local');
```

Comment:
Another `lookup()` here, may I ask you to double-check the documentation? :+1: 

```suggestion
$promise = $resolver->resolve('me.local');
```

Reply:
Opps, I thought you meant just the specific line, but I checked the whole README.md for `lookup`. Thank you @clue 👍